### PR TITLE
adds Router and POST request

### PR DESCRIPTION
**Pull request by MarissaLZ**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,56 +1,78 @@
...
   }
   return (
-  <>
-    <h1>Todo List</h1>
-    <AddTodoForm onAddTodo={addTodo}/>
-    { isLoading ? <p>Loading</p> : <TodoList todoList={todoList} onRemoveTodo={removeTodo}/>}
-  </>
+    <Router>
+      <Routes>
+        <Route path="/" element={<>
```
Comment (oxana-ermolenko) [inline src/App.js:66]: use prop exact in line 66

**Review by oxana-ermolenko — COMMENTED** [review]
Good job!

File: src/App.js (hunk 1 above)
Comment (MarissaLZ) [reply src/App.js:66]: Hi Oxana, I think I read that in React Route V6 the exact prop is not supported.

**Comment by MarissaLZ** [comment]
thank you!

File: src/App.js (hunk 1 above)
Comment (oxana-ermolenko) [reply src/App.js:66]: ok

**Pull request merged**